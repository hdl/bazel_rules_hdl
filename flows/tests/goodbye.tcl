set hello_file [open $::env(INPUT_HELLO) "r"]
set hello_txt [read $hello_file]
close $hello_file
puts $hello_txt

set goodbye_file [open $::env(OUTPUT_GOODBYE) "w"]
puts $goodbye_file $hello_txt
puts $goodbye_file "Goodbye, cruel world!"
close $goodbye_file
