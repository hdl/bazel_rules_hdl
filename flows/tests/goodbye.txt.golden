Hello, TestName, Bazel and OpenROAD!

Goodbye, cruel world!
